[2025-04-06 08:47:36,875] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.mapping_categorical_vars manual__2025-04-06T08:47:20.760699+00:00 [queued]>
[2025-04-06 08:47:36,880] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.mapping_categorical_vars manual__2025-04-06T08:47:20.760699+00:00 [queued]>
[2025-04-06 08:47:36,880] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2025-04-06 08:47:36,880] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2025-04-06 08:47:36,880] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2025-04-06 08:47:36,903] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): mapping_categorical_vars> on 2025-04-06 08:47:20.760699+00:00
[2025-04-06 08:47:36,908] {standard_task_runner.py:52} INFO - Started process 20603 to run task
[2025-04-06 08:47:36,911] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Lead_Scoring_Data_Engineering_Pipeline', 'mapping_categorical_vars', 'manual__2025-04-06T08:47:20.760699+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/Lead_scoring_data_pipeline/.ipynb_checkpoints/lead_scoring_data_pipeline-checkpoint.py', '--cfg-path', '/tmp/tmp3u4onuz3', '--error-file', '/tmp/tmps7dk5rbp']
[2025-04-06 08:47:36,911] {standard_task_runner.py:80} INFO - Job 61: Subtask mapping_categorical_vars
[2025-04-06 08:47:36,956] {task_command.py:371} INFO - Running <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.mapping_categorical_vars manual__2025-04-06T08:47:20.760699+00:00 [running]> on host c6537eab7989
[2025-04-06 08:47:37,008] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Lead_Scoring_Data_Engineering_Pipeline
AIRFLOW_CTX_TASK_ID=mapping_categorical_vars
AIRFLOW_CTX_EXECUTION_DATE=2025-04-06T08:47:20.760699+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-04-06T08:47:20.760699+00:00
[2025-04-06 08:47:37,352] {logging_mixin.py:115} WARNING - /home/mlopsAssignment/Airflow/Dags/Lead_scoring_data_pipeline/utils.py:192 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-04-06 08:47:37,397] {logging_mixin.py:115} WARNING - /home/mlopsAssignment/Airflow/Dags/Lead_scoring_data_pipeline/utils.py:201 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-04-06 08:47:37,436] {logging_mixin.py:115} WARNING - /home/mlopsAssignment/Airflow/Dags/Lead_scoring_data_pipeline/utils.py:210 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-04-06 08:47:38,053] {python.py:173} INFO - Done. Returned value was: None
[2025-04-06 08:47:38,058] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=Lead_Scoring_Data_Engineering_Pipeline, task_id=mapping_categorical_vars, execution_date=20250406T084720, start_date=20250406T084736, end_date=20250406T084738
[2025-04-06 08:47:38,087] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-04-06 08:47:38,155] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
